.github/workflows/build_tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Run Build Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - dev
+  workflow_dispatch:
+
+jobs:
+  py_build_tests:
+    uses: neongeckocom/.github/.github/workflows/python_build_tests.yml@master
+    with:
+      test_manifest: true
+      manifest_ignored: "test/**,scripts/**,CHANGELOG.md,regex/**,vocab/**,**.txt"
+  pip_audit:
+    strategy:
+      max-parallel: 2
+      matrix:
+        python-version: ["3.10"]
```

Comment:
What's the reasoning behind this? It's currently failing, and the two libraries that are failing the check are pinned at the core level, so there's not much to do about the vulnerabilities that it identified.

Reply:
they were all just copy pasted from the hello skill

Reply:
Need a hand excluding them?